.storybook/pages/WireframeDemo/WireframeDemo.tsx

```diff
@@ -0,0 +1,411 @@
+        className={styles['logged-out-page__button-group']}
+        orientation="vertical"
+      >
+        {['Hogwarts', 'Cohorts', 'Demos', 'Admins'].map((buttonText) => (
+          <Button
+            className="w-[14rem]"
+            key={`other-login-${buttonText}`}
+            onClick={onLogin}
+            variant="primary"
+          >
+            {buttonText}
+          </Button>
+        ))}
+      </ButtonGroup>
+    </div>
+
+    <div className="flex flex-col items-center gap-2">
+      <img
+        alt="placeholder for decorative illustration"
+        className="w-16 mb-2"
+        src={PlaceholderImage}
+      />
+      <Text>Don't have an account?</Text>
+      <Button onClick={onLogin} variant="link">
+        <Icon name="circle" purpose="decorative" />
+        Create an account
+      </Button>
+    </div>
+
+    <GlobalFooter />
+  </div>
+);
+
+const StudentTab = ({
+  studentName,
+  active = false,
+}: {
+  studentName: string;
+  active?: boolean;
+}) => (
+  <div
+    className={clsx(
+      styles['student-tab'],
+      active && styles['student-tab--active'],
+    )}
+  >
+    <div className={clsx(styles['watch-page__avatar'], 'w-10 h-10')}>
+      <Text size="sm">{studentName.slice(0, 1)}</Text>
+    </div>
+    <Text className="!mt-2 truncate" size="xs">
+      {studentName}
+    </Text>
+  </div>
+);
+
+const WatchPage = ({ onLogout }: { onLogout: () => void }) => {
+  const responseTextareaId = useUID();
+
+  const studentGroupOptions = [
+    {
+      key: '1',
+      label: 'All students',
+    },
+    {
+      key: '2',
+      label: 'Not grouped',
+    },
+  ];
+
+  const [selectedOption, setSelectedOption] = useState(studentGroupOptions[0]);
+  const [canSubmitResponse, setCanSubmitResponse] = useState(false);
+
+  const handleTextareaChange = (event: any) => {
+    if (event.target.value?.length >= 3 && !canSubmitResponse) {
+      setCanSubmitResponse(true);
+    } else if (event.target.value?.length < 3 && canSubmitResponse) {
+      setCanSubmitResponse(false);
+    }
+  };
+
+  return (
+    <div className="">
+      <div className={styles['watch-page__main-section']}>
+        <div className="ml-10 mr-8 flex gap-4">
+          <Button onClick={onLogout} variant="secondary">
+            <Icon name="arrow-back" purpose="informative" title="go back" />
+          </Button>
+          <div>
+            <Text className="mb-2">Playing reflections in response to:</Text>
+            <Heading as="h1" size="h3">
+              What's something in your life, big or small, that you're proud of?
+              Why are you proud of it?
+            </Heading>
+          </div>
+        </div>
+
+        <div className={styles['watch-page__carousel']}>
+          <div className="w-[10.5rem]">
+            <Dropdown
+              aria-label="student groups"
+              buttonText={selectedOption.label}
+              // TODO: why isn't the selected styling showing up when selecting a new option?
```

Comment:
I have a ticket for investigating the [dropdown selection style being weird](https://czi-tech.atlassian.net/browse/EDS-733).

Reply:
Think the new dropdown work can also help with this